bdd-generator/src/main/java/com/epam/test_generator/services/StepService.java

```diff
@@ -127,7 +128,7 @@ public StepDTO updateStep(Long projectId, Long suitId, Long caseId, Long stepId,
         StepSuggestion oldStepSuggestion = stepSuggestionService.getStepSuggestion(projectId, oldStep);
         Step newStep = stepTransformer.updateFromDto(stepUpdateDTO, oldStep);
         StepSuggestion newStepSuggestion = stepSuggestionService.getStepSuggestion(projectId, newStep);
-        stepDAO.save(newStep);
+        stepSuggestionDAO.save(newStepSuggestion);
```

Comment:
I doubt this call is required, please double check.

Reply:
I checked call this method. It really isn't needed. And I also deleted two useless calls.